code/src/App.js

```diff
@@ -1,9 +1,37 @@
-import React from 'react'
+import React, { useState, useEffect } from "react";
+import { BrowserRouter, Switch, Route, Redirect } from "react-router-dom";
+import { API_URL } from "utils/url";
+
+import MovieList from "components/MovieList";
+import MovieDetail from "components/MovieDetail";
+import PageNotFound from "components/PageNotFound";
+import Loading from "components/Loading";
 
 export const App = () => {
+  const [movie, setMovie] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    fetch(API_URL)
+      .then((res) => res.json())
+      .then((data) => {
+        setMovie(data.results);
+      })
+      .finally(() => setLoading(false));
+  }, []);
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <>
+      {loading && <Loading />}
```

Comment:
Nice detail with the loading item! We tried to slow down the network to slow 3G to be able to see it when loading the main page but we couldn't, only for the movie details page. We were unsure if it is infront of the rest of the content on the main page. Can you make it visible?